qa/tasks/cephfs/cephfs_test_case.py

```diff
@@ -175,6 +181,9 @@ def tearDown(self):
         for i, m in enumerate(self.mounts):
             m.client_id = self._original_client_ids[i]
 
+        if self.vol_created:
+            self.fs.rm(self.volname)
```

Comment:
```suggestion
        if self.fs is not None:
            self.fs.rm()
            self.fs = None
```

Reply:
Addressed in https://github.com/ceph/ceph/pull/36368